axelrod/strategies/titfortat.py

```diff
@@ -554,3 +554,48 @@ def reset(self):
     def __repr__(self):
 
         return "%s: %s" % (self.name, round(self.rate, 2))
+
+class SpitefulTitForTat(Player):
+    """
+    A player starts by cooperating and then mimics the previous action of the
+    opponent until opponent defects twice in a row, at which point player always defects
+
+    Names:
+
+    - Spiteful Tit For Tat
+    """
+
+    name = 'Spiteful Tit For Tat'
+    classifier = {
+        'memory_depth': 2,
```

Comment:
The memory_depth is `float('inf')` because of the eternal defection that can kick in.